docs/Dual_Carriage.md

```diff
@@ -0,0 +1,98 @@
+This document provides **general** information on configuring and
+calibrating a standard IDEX printer. Klipper has support for very
+complex IDEX configurations, with multiple hotends and extruders on a
+single carriage, independent or grouped cooling fans, and much more,
+which are not covered in this document.
+
+# General configuration
+Start with studying, and possibly using parts of the [example config](../config/sample-idex.cfg)
+for IDEX printers. It has some distinct items that you may or may not
+wish to use.
+* Definition for `dual_carriage` on the X-axis, which you can change to
+be on the Y axis if that is how your printer is configured.
+* Sample macros for parking and switching toolheads. You should check
+these to make sure they make sense for your setup and update the
+position values with those that match your printer
+* Configuration for multiple part cooling fans. The `[fan]` module in
```

Comment:
Could you also add the following about a `[display]` section:
* If you have a `[display]` section of a st7920 or uc1701 type, set the `display_group: _multiextruder_16x4` to get a dual-extruder summary view.

Reply:
> My comment and proposed solution was to have a `[fan]` section but then override the M106 & M107 gcodes. This way the display will show the "active extruder's" fan speed.

how does the system know which fan is tied to the active extruder and switch that? how are you configuring the second fan? can you send me your config?



> Could you also add the following about a `[display]` section:
> 
> * If you have a `[display]` section of a st7920 or uc1701 type, set the `display_group: _multiextruder_16x4` to get a dual-extruder summary view.

I'll consider it. Can you send me pictures of what that looks like and do you know what to do about other displays or are those the common display types?

Reply:
Those are the common 12864 and 12864 mini display types. I have a repurposed CR10 display which is a Reprapdiscount 12864 clone without an SD card reader.

I'll attach the T0/T1 macros.  they are heavily based on the sample ones.

Reply:
It might be better to say if it's a common 12864 or 12864 mini display.
Here is a picture:

![image](https://user-images.githubusercontent.com/1309160/133685432-cc3b0d96-2ddf-4f6e-91a4-f9968754a921.png)